examples/spawn-and-move/src/actions.cairo

```diff
@@ -30,8 +28,15 @@ mod actions {
         Position, Moves, Direction, Vec2, PlayerConfig, PlayerItem, ServerProfile
     };
     use dojo_examples::utils::next_position;
-    #[cfg(feature: 'something')]
-    use dojo_examples::something::{ISomethingDispatcher, ISomethingDispatcherTrait};
+
+    // Featues can be used on modules, structs, trait and `use`. Not inside
+    // a function.
+    #[cfg(feature: 'dungeon')]
+    use dojo_examples::dungeon::{IDungeonDispatcher, IDungeonDispatcherTrait};
+    #[cfg(feature: 'dungeon')]
+    use armory::Flatbow;
+    #[cfg(feature: 'dungeon')]
+    use bestiary::RiverSkale;
```

Comment:
@remybar made a comment and we will update to ensure any library imported can be placed under a different namespace if the user wants to.

So currently what happens, is that the model is imported, and at the compilation time, the namespace of the current package is used instead of the default namespace of the library.

For this reason, if someone is forcing the namespace like `#[dojo::model(namespace: "nm1")]`, hence the namespace of the model **even if imported elsewhere** will be `nm1`. This is due to how we define the namespace during the plugin expansion phase.

Reply:
> incase we are importing from a library we don't need to specify its namespace?

 Just updated the description of the PR with an example. 👍